### ISSUE TITLE
Global domain size and meaning

### ISSUE THREAD

**Issue by alpha358** [creiser/kilonerf#10]
Any suggestions on how to estimate the global domain size?
- It defined [at this line](https://github.com/creiser/kilonerf/blob/ac0fdba0963d1a05568bf192b7a8a534156eaa20/utils.py#L268).
- Could it be understood as the spatial region where ray tracking integrals of color value will be computed?
     - Then we just need to guess the 3D bounding box for the region of interest?

**Comment by creiser** [comment]
Yes, this describes a bounding box that encompasses the whole scene. The grid of networks will only live in that bounding box. The size of the bounding box could be automatically inferred with the SfM algorithm (e.g. COLMAP) that you have to run anyhow for estimating the camera poses of the training views. SfM outputs a point cloud. The size of the bounding box must be such that it contains the entire point cloud.

Analogically, you have to adjust the "near" and "far" hyperparameters such that the entire scene is sampled. This can also be done based on the sparse point cloud from SfM.
